.clang-format

```diff
@@ -0,0 +1,10 @@
+AlignTrailingComments: 'false'
```

Comment:
This is looking very good.
This file can lead to unexpected reformatting.
Can you please move this .clang-format file into the /sndio/ folder?

Reply:
Fixed in #878